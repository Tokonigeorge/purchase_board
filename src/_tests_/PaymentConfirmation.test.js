/**
 * @jest-environment jsdom
 */
import * as React from "react";

import renderer from "react-test-renderer";
import PaymentConfirmation from "../components/paymentConfirm/PaymentConfirmation";

describe("It renders Sucessfully", () => {
  test("eventHandler called on click", () => {
    const tree = renderer.create(<PaymentConfirmation />);
    //create a snapshot for the component
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
