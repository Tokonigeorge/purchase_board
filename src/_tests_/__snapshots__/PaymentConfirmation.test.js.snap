// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It renders Sucessfully eventHandler called on click 1`] = `
<div
  className="bg-white w-full rounded-md mx-8 md:mx-0 py-8 px-12 md:py-16 md:px-24"
>
  <div
    className="shadow-2xl rounded-full w-12 h-12 flex justify-center items-center  m-auto"
  >
    <span
      className="text-center"
    >
      <svg
        fill="none"
        height="20"
        viewBox="0 0 48 35"
        width="30"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          clipRule="evenodd"
          d="M46.5041 6.7984L19.3601 33.7552C17.9681 35.1376 15.7641 35.1376 14.3721 33.7552L1.49615 20.968C0.104148 19.5856 0.104148 17.3968 1.49615 16.0144C2.88815 14.632 5.09215 14.632 6.48415 16.0144L16.8661 26.3248L41.5161 1.8448C42.9081 0.462399 45.1121 0.462399 46.5041 1.8448C47.8961 3.2272 47.8961 5.416 46.5041 6.7984Z"
          fill="#6FCF97"
          fillRule="evenodd"
        />
      </svg>
    </span>
  </div>
  <p
    className="text-purp font-bold text-3xl pt-2"
  >
    Purchase Completed
  </p>
  <p
    className="text-gray2 font-light text-sm pt-5 pr-12 mb-5"
  >
    Please check your email for details concerning this transaction
  </p>
  <a
    className="text-btnGradOne underline text-sm"
    href="#"
  >
    Return home
  </a>
</div>
`;
